libs/core/algorithms/include/hpx/parallel/algorithms/reduce_deterministic.hpp

```diff
@@ -396,10 +397,67 @@ namespace hpx::parallel {
             static constexpr T sequential(ExPolicy&& policy, InIterB first,
                 InIterE last, T_&& init, Reduce&& r)
             {
+                // TODO: abstract initializing memory
+                hpx::parallel::detail::rfa::RFA_bins<T_> bins;
+                bins.initialize_bins();
+                std::memcpy(hpx::parallel::detail::rfa::__rfa_bin_host_buffer__,
```

Comment:
Please don't use symbols that start with `__`. Those are reserved for the standard library implementation.